src/vulkan/gpu.c

```diff
@@ -2562,6 +2591,27 @@ static const struct pl_sync *vk_sync_create(const struct pl_gpu *gpu,
         VK(vk->vkGetSemaphoreFdKHR(vk->dev, &finfo, &sync->signal_handle.fd));
     }
 #endif
+#ifdef VK_HAVE_WIN32
+    VkExternalSemaphoreHandleTypeFlagBits handle_type = 0;
```

Comment:
`handle_type` is a name clash, suggest `win32_handle_type` - or possibly just a ternary in `handle_info`, since it doesn't seem to be used anywhere else (except the `if`)